applications/adv_networking_bench/adv_networking_bench_default_tx_rx.yaml

```diff
@@ -77,14 +77,15 @@ advanced_network:
             # ipv4_len: 1050
 
 bench_rx:
-  port_name: "rx_port"    # Name of rx_port from the advanced_network config
+  interface_name: "rx_port"    # Name of rx_port from the advanced_network config
   gpu_direct: true        # Set to true if using a GPU region for the Rx queues.
   split_boundary: false   # Whether header and data is split (Header to CPU)
   batch_size: 10240
   max_packet_size: 1064
   header_size: 64
 
 bench_tx:
+  interface_name: "tx_port"    # Name of tx_port from the advanced_network config
```

Comment:
Thank you that's great.

We need to update the docs/tutorials accordingly. Happy to do it myself